projects/1.20/assets/thalassophobia/thalassophobia/lang/zh_cn.json

```
@@ -0,0 +1,213 @@
+  "entity.thalassophobia.megapiranha": "巨食人鱼",
+  "entity.thalassophobia.megastar": "巨型海星",
+  "entity.thalassophobia.megastar_natural_spawn": "巨型海星",
+  "entity.thalassophobia.megastar_spawn": "巨型海星",
+  "entity.thalassophobia.mosasaur": "沧龙",
+  "entity.thalassophobia.panderodus": "潘氏鱼",
+  "entity.thalassophobia.plesiosaur": "蛇颈龙",
+  "entity.thalassophobia.reaverfish": "掠夺鱼",
+  "entity.thalassophobia.riffhead": "礁石头鱼",
+  "entity.thalassophobia.sea_bird": "海鸟",
+  "entity.thalassophobia.sea_bird_spawn": "海鸟",
+  "entity.thalassophobia.sea_phantom": "海洋幻影",
+  "entity.thalassophobia.sea_sludge": "海泥",
+  "entity.thalassophobia.sea_sludge_natural_spawn": "海泥",
+  "entity.thalassophobia.sea_sludge_spawn": "海泥",
+  "entity.thalassophobia.shark": "鲨鱼",
+  "entity.thalassophobia.sonar_boom": "声呐爆炸",
+  "entity.thalassophobia.sunfish": "翻车鱼",
+  "entity.thalassophobia.tusoteuthis": "托斯特巨鱿",
+  "entity.thalassophobia.vertebrat": "脊椎鱼",
+  "entity.thalassophobia.xiphactinus": "剑射鱼",
+  "entity.thalassophobia.zapfin": "电鳍鱼",
+  "gamerule.enableAlphaReaverfishSpawning": "生成阿尔法掠夺鱼",
+  "gamerule.enableAlphaReaverfishSpawning.description": "是否生成阿尔法掠夺鱼",
+  "item_group.thalassophobia.thalassophobia": "深海恐惧症",
+  "item.thalassophobia.alpha_fang": "阿尔法尖牙",
+  "item.thalassophobia.alpha_reaverfish_spawn_egg": "阿尔法掠夺鱼生成蛋",
+  "item.thalassophobia.ammonita_spawn_egg": "菊石生成蛋",
+  "item.thalassophobia.anglerfish_bucket": "鮟鱇鱼桶",
+  "item.thalassophobia.anglerfish_spawn_egg": "鮟鱇鱼生成蛋",
+  "item.thalassophobia.archelon_spawn_egg": "古海龟生成蛋",
+  "item.thalassophobia.baby_colossoteus_spawn_egg": "幼年巨兽龙生成蛋",
+  "item.thalassophobia.bathotrap_natural_spawn_spawn_egg": "深渊陷阱生成蛋",
+  "item.thalassophobia.bathotrap_spawn_egg": "深渊陷阱生成蛋",
+  "item.thalassophobia.bathotrap_spawn_spawn_egg": "深渊陷阱生成蛋",
+  "item.thalassophobia.black_pearl": "黑珍珠",
+  "item.thalassophobia.cnidaria_spawn_egg": "刺胞动物生成蛋",
+  "item.thalassophobia.colossoteus_spawn_egg": "成年巨兽龙生成蛋",
+  "item.thalassophobia.cooked_crab": "熟螃蟹",
+  "item.thalassophobia.cooked_fish_meat": "熟鱼肉",
+  "item.thalassophobia.cooked_glow_fish": "熟发光鱼肉",
+  "item.thalassophobia.cooked_sea_bird": "熟海鸟肉",
+  "item.thalassophobia.crab_bucket": "螃蟹桶",
+  "item.thalassophobia.crab_natural_spawn_spawn_egg": "螃蟹生成蛋",
+  "item.thalassophobia.crab_spawn_egg": "螃蟹生成蛋",
+  "item.thalassophobia.crab_spawn_spawn_egg": "螃蟹生成蛋",
+  "item.thalassophobia.dead_archelon_spawn_egg": "古海龟尸体生成蛋",
+  "item.thalassophobia.dead_archelon_spawn_spawn_egg": "古海龟尸体生成蛋",
+  "item.thalassophobia.deep_sea_amulet": "深海护符",
+  "item.thalassophobia.deep_shield": "深海盾牌",
+  "item.thalassophobia.dunkleosteus_spawn_egg": "邓氏鱼生成蛋",
+  "item.thalassophobia.echo_shell": "回声壳",
+  "item.thalassophobia.electrophorus_spawn_egg": "电鳗生成蛋",
+  "item.thalassophobia.eurypterid_bucket": "翼肢鲎桶",
+  "item.thalassophobia.eurypterid_spawn_egg": "翼肢鲎生成蛋",
+  "item.thalassophobia.fin": "鱼鳍",
+  "item.thalassophobia.fish_scale": "鱼鳞",
+  "item.thalassophobia.fish_stew": "鱼汤",
```

```suggestion
  "item.thalassophobia.fish_stew": "鱼肉煲",
```
和原版的蘑菇煲（Mushroom Stew）、兔肉煲（Rabbit Stew）统一